src/iotjs.cpp

```diff
@@ -98,7 +98,9 @@ static void CleanupModules() {
 static bool RunIoTjs(JObject* process) {
   // Evaluating 'iotjs.js' returns a function.
 #ifndef ENABLE_SNAPSHOT
-  JResult jmain = JObject::Eval(String(iotjs_s, iotjs_l), false);
+  iotjs_string_t code = iotjs_string_create_with_buffer((char*)iotjs_s,
```

Comment:
In C pointers are usually preferred, iotjs_string_t* rather than iotjs_string_t. This could remain though, but looks a bit unusual for me.